hopsworks-common/src/main/java/io/hops/hopsworks/common/dao/AbstractFacade.java

```diff
@@ -66,6 +66,7 @@ public AbstractFacade(Class<T> entityClass) {
 
   public void save(T entity) {
     getEntityManager().persist(entity);
+    getEntityManager().flush();
```

Comment:
Do you need this in the abstract facade? I think it is better to override it in the facade that needs flush.